extensions/sticky/src/PinStickiedDiscussionsToTop.php

```diff
@@ -58,7 +58,7 @@ public function __invoke(DatabaseSearchState $state, SearchCriteria $criteria):
                 // Add the bindings manually (rather than as the second
                 // argument in orderByRaw) for now due to a bug in Laravel which
                 // would add the bindings in the wrong order.
-                $q->selectRaw('(is_sticky and not exists ('.$read->toSql().') and last_posted_at > ?) as is_unread_sticky', array_merge($read->getBindings(), [$state->getActor()->marked_all_as_read_at ?: 0]));
+                $q->selectRaw('(is_sticky and not exists ('.$read->toSql().') and last_posted_at > ?) as is_unread_sticky', array_merge($read->getBindings(), [$state->getActor()->marked_all_as_read_at ?: '1970-01-01 00:00:00']));
```

Comment:
Is there no constant somewhere for epoch time?

Reply:
changed to:

```php
$epochTime = (new DateTime('@0'))->format('Y-m-d H:i:s');
```